scripts/jenkins/ardana/ansible/post-deploy-checks.yml

```diff
@@ -0,0 +1,30 @@
+- name: Take snapshot of modified/deleted rpm-owned files post deployment
+  shell: rpm -Va | grep -vxFf "{{ pre_deploy_rpm_verification }}"
```

Comment:
I think you might need `become: true` for this, at least on my system I see a lot of "missing [...] (Permission denied)" when I run this without sudo.

Reply:
@cmurphy is right, this can produce false `missing c` files in the report, just because rpm cannot read the directory

Reply:
Good catch thanks, that's the bit I was unable to test properly yet due to the Horizon issue. Fixed now.